marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.smart-mastering/matcher-impl/options-impl.xqy

```diff
@@ -424,71 +424,78 @@ declare function opt-impl:compile-match-options(
           $rule-set/matchRules
         else
           $rule-set
+      let $multi-struct-prop-multi-value-map :=
+        if ($is-complex-rule) then
+          opt-impl:multi-struct-prop-multi-value-map($match-rules, $target-entity-def)
```

Comment:
This is throwing an error, likely due to a rebase. This variable should be `$target-entity-type-def` now.

Reply:
Yes, from rebase. Fixed.